Cloud Application Manager/Core Concepts/workspaces-and-collaboration.md

```diff
@@ -159,9 +189,12 @@ Also you may give view access to make an instance available for others to use, s
 
 #### Sharing Providers
 
-Sharing providers has its benefits. You can give view access to company-approved providers and let users deploy to that particular provider. When teams deploy to a shared provider, you can track org-wide usage and compliance cohesively. Provider accounts can be shared only in the Enterprise Edition.
+Sharing providers has its benefits. You can give view access to company-approved providers and let users deploy to that particular provider. When teams deploy to a shared provider, you can track org-wide usage and compliance cohesively.  
+Provider accounts can be shared only in the Enterprise Edition.
 
-### Sharing with people or groups across Organizations
+### Federated Organizations
```

Comment:
Leave it as it was

Reply:
In CAM-11115 suggest Sharing with people or groups across Organizations
![image](https://user-images.githubusercontent.com/42206552/58336910-f3858300-7e44-11e9-9883-4230dd2a0411.png)